lxd/instance/drivers/driver_qemu.go

```diff
@@ -4766,21 +4747,6 @@ func (d *qemu) updateDevices(removeDevices deviceConfig.Devices, addDevices devi
 			d.logger.Error("Device stop validation failed", logger.Ctx{"devName": dd.Name, "err": err})
 		}
 
-		// If a device was returned from deviceLoad even if validation fails, then try and stop and remove.
```

Comment:
We still need this section.

Reply:
I brought it back here and in `lxd/instance/drivers/driver_lxc.go`